samples/crypto/persistent_key_usage/README.rst

```diff
@@ -1,36 +1,34 @@
 .. _crypto_persistent_key:
 
-Crypto: AES CTR
-###############
+Crypto: Persistent key storage
+##############################
 
 .. contents::
    :local:
    :depth: 2
 
+The persistent key sample shows how to generate a persistent key using the Platform Security Architecture (PSA) APIs.
```

Comment:
```suggestion
The persistent key sample shows how to generate a persistent key using the Platform Security Architecture (PSA) API.
```
For consistency with the other instances

Reply:
Multiple APIs may be used to generate a key. I don't think the removal of s in APIs is neccesary...